workers/unity/Assets/Playground/Config/PlayerTemplate.cs

```diff
@@ -1,54 +1,56 @@
+using System;
 using System.Collections.Generic;
-using Generated.Improbable;
+using System.Linq;
+using Generated.Improbable.PlayerLifecycle;
+using Generated.Improbable.Transform;
+using Generated.Playground;
 using Improbable.Gdk.Core;
-using Improbable.Gdk.Legacy;
+using Improbable.Worker.Core;
 
 namespace Playground
 {
     public static class PlayerTemplate
     {
-        private static readonly WorkerRequirementSet GameLogicSet = new WorkerRequirementSet
-        {
-              AttributeSet  = {UnityGameLogic.WorkerAttributeSet}
-        };
+        private static readonly string GameLogicAttribute = UnityGameLogic.WorkerType;
 
-        private static readonly WorkerRequirementSet AllWorkerSet = new WorkerRequirementSet {
-            AttributeSet = { UnityGameLogic.WorkerAttributeSet, UnityClient.WorkerAttributeSet }
-        };
+        private static readonly List<string> AllWorkerAttributes =
+            new List<string> { UnityGameLogic.WorkerType, UnityClient.WorkerType };
 
         public static Entity CreatePlayerEntityTemplate(List<string> clientAttributeSet,
             Generated.Improbable.Vector3f position)
         {
-            var clientSet = new WorkerRequirementSet(new Improbable.Collections.List<WorkerAttributeSet>()
+            var clientAttribute = clientAttributeSet.First(attribute => attribute != UnityClient.WorkerType);
```

Comment:
I assume that this works but can somebody explain to me the semantics of the callerAtrributeSet of a command here?

Reply:
When a command is received, you get the attribute set of the worker that sent the command. With the way the current GDK project is set up, the attribute set of a client will have the following: 

`UnityClient` `UnityClient-<id>` 

So this line basically says: give me the attribute that is not the generic `UnityClient` attribute - the attribute unique to that client

Reply:
Ah nice! Thanks for explaining. :)